### PR TITLE
Fix date picker bug

### LINKED ISSUE #2127: Start time on running time entry and editing existing time entry is displayed as UTC
When editing the Start time, by clicking on the Start time in the wheel menu, of a running or stopped time entry.

The time on the wheel is displayed as UTC.

image

### PR DESCRIPTION
**Pull request by arturs105**
Closes #2127

This is a workaround for the issue, because i couldn't find out why it was working for the end time, but not for the start time. More in-depth explanation can be found in the issue.

Squash whenever.

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/ViewModels/EditDurationViewModel.cs [hunk 1]
```diff
@@ -184,6 +187,7 @@ private void editStartTime()
             }
             else
             {
+                AboutToEditStartTime.Raise(this);
```
Comment (simonrozsival) [inline Toggl.Foundation.MvvmCross/ViewModels/EditDurationViewModel.cs]: We should test this.

**Comment by simonrozsival** [comment]
What is the status of this PR? We had an internal discussion, but I am not sure what was the final decision - will you rewrite this so it uses observables instead of events? Or will it stay this way?

**Comment by arturs105** [comment]
Yeah, i've already rewritten the thing locally, i just need to test it out before pushing.

**Comment by simonrozsival** [comment]
OK 👌

File: Toggl.Daneel/ViewControllers/EditDurationViewController.cs [hunk 2]
```diff
@@ -200,6 +204,13 @@ public override void ViewDidLayoutSubviews()
             adjustHeight();
         }
 
+        protected override void Dispose(bool disposing)
```
Comment (willsb) [inline Toggl.Daneel/ViewControllers/EditDurationViewController.cs]: Please dispose your stuff here

**Pull request merged**